pendulum_msgs/msg/RttestResults.msg

```diff
@@ -0,0 +1,11 @@
+builtin_interfaces/Time stamp
+
+pendulum_msgs/JointCommand command
+pendulum_msgs/JointState state
```

Comment:
Yeah, I did this just because it seems to be used other places in the message definitions when a sub-message from out of the package is being used (like builtin_interfaces). I think I'll omit the package name from messages that are included in this package but not messages that come from outside of the package.